### PR TITLE
[DBAAS-188] security related changes 

### PR DESCRIPTION
**Pull request by tchughesiv**
## Description
Improvements recommended by prod sec team. Unable to add probes for kube rbac proxy container due to following known issue.
https://github.com/kubernetes/kubernetes/issues/43784

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist (Definition of Done)

- [x] All acceptance criteria specified in JIRA or in issue number have been completed
- [ ] Unit tests added that prove the fix is effective or the feature works 
- [ ] Documentation added for the feature
- [x] all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

**Review by jeremyary — APPROVED** [review]
lgtm Tommy, thanks!

File: bundle/manifests/dbaas-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -504,6 +504,12 @@ spec:
                 - containerPort: 8443
                   name: https
                 resources: {}
+                securityContext:
+                  allowPrivilegeEscalation: false
+                  capabilities:
+                    drop:
+                    - NET_RAW
```
Comment (JoshuaMulliken) [inline bundle/manifests/dbaas-operator.clusterserviceversion.yaml:511]: I see that you are only dropping `NET_RAW` are all of the remaining default caps required?

Ref: <https://github.com/moby/moby/blob/master/oci/caps/defaults.go#L6-L19>

File: bundle/manifests/dbaas-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -543,8 +549,10 @@ spec:
                     memory: 100Mi
                 securityContext:
                   allowPrivilegeEscalation: false
+                  capabilities:
+                    drop:
+                    - NET_RAW
```
Comment (JoshuaMulliken) [inline bundle/manifests/dbaas-operator.clusterserviceversion.yaml:554]: Same as https://github.com/RHEcosystemAppEng/dbaas-operator/pull/108/files#r800699331

**Comment by tchughesiv** [comment]
@JoshuaMulliken i've dropped all caps

**Review by JoshuaMulliken — APPROVED** [review]
Looks great! I am particularly happy with the addition of a strong securityContext for the workloads!

**Comment by jeremyary** [comment]
thanks for pitching in @JoshuaMulliken !

**Pull request merged**